FlightTelemetry_2_1_2020/FlightTelemetry_2_1_2020.ino

```diff
@@ -0,0 +1,150 @@
+#include <Wire.h>

```

Comment:
There's no need to duplicate the whole file - git does version history tracking for you, and it is bad style to clutter the repository with multiple versions of essentially the same code.  Instead it is better to have a single copy and use C preprocessor macros to allow for easy toggling between build versions.

Reply:
Agreed, I was being stupid and made a whole new .ino file to start making edits on. I'll consolidate the changes for a push request on master or that other branch you made for me

Reply:
This branch is fine, you already made a pull request.  Just merge the files and make another commit on this branch.